Add new blog post (#680)
vmware-tanzu/cartographer: site/content/posts/contributor-of-the-month.md

---
title: "Introducing the Contributor of the Month initiative"
slug: contributor-of-the-month
date: 2022-03-01
author: Cartographer Team
authorLocation: https://github.com/vmware-tanzu/cartographer/blob/main/MAINTAINERS.md
image: /img/posts/contributor-of-the-month/Contributor-of-the-month-v3.png
excerpt: "A monthly program to reward contributions"
tags: ['community']
---

> Since October 2021, when the Cartographer project was made public, we have been very fortunate to count with individuals who have helped improve the codebase and docs while also telling the Cartographer story in conferences, live streams and other forms of content. All of them have done this out of intrinsic motivation, without expecting any reward or public recognition and we are forever grateful to them for this.

Reviewer: ```suggestion
Since October 2021, when the Cartographer project was made public, we have been very fortunate to count on individual contributors who have helped improve the codebase and docs while also telling the Cartographer story at conferences, in live streams and other forms of content. All of them have done this out of an intrinsic motivation, without expecting any reward or public recognition. We are forever grateful to them for this.
```

Nevertheless, we would like to fill in the [extrinsic motivation](https://www.rochester.edu/emerging-leaders/understanding-intrinsic-and-extrinsic-motivation/#:~:text=Intrinsic%20motivation%20involves%20performing%20a,punishment%20or%20receiving%20a%20reward) void by rolling out a small initiative called **Contributor of the Month**

**How does it work?**

We plan to recognize contributors in three different categories:

* Code contributor (merged PRs)
* Content creator (tweets/blog posts/screencasts/presentations/podcast)
* Project and community supporter (community meetings participation/Slack interactions)

> Code contributions will be tracked using the `contributor shoutouter` program located in [this repo](https://github.com/vmware-tanzu/community-engagement/tree/main/shoutouter) while content and community support will be tracked down by the Cartographer community team.

Reviewer: ```suggestion
Code contributions will be tracked using the `contributor shoutouter` program located in [this repo](https://github.com/vmware-tanzu/community-engagement/tree/main/shoutouter) while content and community support will be tracked by the Cartographer community team.
```

> Code contributions will be tracked using the `contributor shoutouter` program located in [this repo](https://github.com/vmware-tanzu/community-engagement/tree/main/shoutouter) while content and community support will be tracked down by the Cartographer community team.

Reviewer: Have we got a list of who is in the "Cartographer community team?" (I'm guessing it's you David :) ) - Is it something we can link to? (perhaps something we should do as we pull the Docs site out to cartographer-site? Make sure when we refer to teams and individuals, there's a directory that folks can refer to, know who to reach out to?

Author: You're right, maybe a TEAMS file like Paketo has: https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md

> As a little token of appreciation, contributors will receive branded Cartographer swag that we hope you find useful and conscious.

Reviewer: ```suggestion
As a little token of appreciation, contributors will receive Cartographer branded, sustainable swag that we hope you find useful.
```
Just not sure folks would know what you mean by 'conscious'?

Author: Yeah, it's more 'environmentally responsible', or even better, 'sustainable'. I'm not sure how t-shirts would fall into that category but stickers not, but will keep refining swag. The goal here was to keep it generic so we can iterate on swag without having to change the post

Also, public shoutouts will be done at the first [Community Meeting](https://docs.google.com/document/d/1HwsjzxpsNI0l1sVAUia4A65lhrkfSF-_XfKoZUHI120/edit?usp=sharing) of the following month and on the [@OssCartographer](https://twitter.com/OssCartographer) Twitter account.

> 
> 
> 
> 
> 
> 

Reviewer: ```suggestion
```

Just so that 'reformat code' is more consistent

## Let's build together! 

> Every contribution, every form of support will count, then make sure to check out the following resources to find  opportunities to help build the Cartographer project:

Reviewer: ```suggestion
Every contribution in every form will count, so make sure to check out the following resources for opportunities to help build the Cartographer project:
```
